code/socialDistribution/templatetags/post/post_card.py

```diff
@@ -22,9 +22,13 @@ def post_card(post, author):
         post_type = 'inbox'
     else:
         post_type = 'local'
-    
-    is_liked, likes = get_post_like_info(post, author)
-    like_text = get_like_text(is_liked, likes)
+    try:
+        is_liked, likes = get_post_like_info(post, author)
+        like_text = get_like_text(is_liked, likes)
+    except:
+        is_liked = False
+        likes = 0
+        like_text = ''
```

Comment:
Don't think this try-catch is necessary as get_post_like_info catches all exceptions within the function.